docs/api.md

```diff
@@ -793,7 +835,8 @@ DELETE /api/channels/5678/pins/1234
   "flair": string | null,
 
   "online": boolean,
-  "permissions": int, // Bitfield generated by ORing the user's role permissions
+  "roles": array, // Array of string IDs for each role the user has, not including "user" or "everyone",
+  "permissions": object, // Map of computed permissions the user has
```

Comment:
Can you return this on `/api/users/:id/permissions`? I don't see much point in every single user serialized anywhere having this computed property on it.